docs/src/components/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -227,7 +228,7 @@ DocsPageWrapper.propTypes = {
 
 DocsPageWrapper.defaultProps = {
   blurb: null,
-  designSubpage: null,
+  designSubpage: <DesignPlaceholderPage wrapped />,
```

Comment:
Hmm won't this only be evaluated a single time which means it will be the same instance in each case? I feel like that would be bad and will confuse the reconciler

Reply:
Ah really? It just seemed like the most sensible, React way to do it?

Reply:
Yeah, but I wonder if it's a good idea

```
DocsPageWrapper.defaultProps = {
  designSubpage: React.createElement(DesignPlaceholderPage, { wrapped: true }, null),
};
```

Which means that every `DocsPageWrapper` will have the exact same instance of the element. I'm unsure if React relies on the fact that each element instance will only exist in a single place in the view hierarchy

Reply:
Just been chatting to George and we're going to do it a different way than this anyway, so I'll resolve your comment